src/frontpage/components/Articles/MainArticle.tsx

```diff
@@ -1,23 +1,25 @@
 import React from 'react';
 
-import { routes } from 'articles/components/ArticlesRouter';
 import { IArticle } from 'articles/models/Article';
-import Markdown from 'common/components/Markdown/index';
+import Markdown from 'common/components/Markdown';
 import ResponsiveImage from 'common/components/ResponsiveImage';
+import { getArticleUrl } from 'core/appUrls';
 import { Link } from 'core/components/Router';
 
 import style from './articles.less';
 
 const MainArticle = ({ heading, image, ingress, id }: IArticle) => {
   return (
-    <Link to={routes.detail + id}>
-      <div className={style.articleContainer}>
-        <ResponsiveImage image={image} size="sm" type="article" />
-        <div>
-          <h2>{heading}</h2>
-          <Markdown source={ingress.replace(/#[^\s#]/g, (match) => `# ${match.slice(-1)}`)} />
+    <Link {...getArticleUrl(id)}>
+      <a>
+        <div className={style.articleContainer}>
+          <ResponsiveImage image={image} size="sm" type="article" />
+          <div>
+            <h2>{heading}</h2>
+            <Markdown source={ingress.replace(/#[^\s#]/g, (match) => `# ${match.slice(-1)}`)} />
```

Comment:
What does this regex-replace do?

Reply:
I think it re-writes headings in markdown:
``` markdown
// from
#heading 
// to
# heading
```

Something with the markdown parsers for OW4 and OWF not handling 'wrong markdown' in the same way. I think OW4 renders it as if it were written correctly (with the space), while OWF handles it as a regular string and not a header.
You'll have to look at #320

Reply:
There is an isskje with the Markdown used on OW4-articles not having the same markdown-standard as our Markdown-library here, so this ensures that:
# Titles appear like this
#Instead of this